bibliopixel/layout/matrix.py

```diff
@@ -16,8 +16,8 @@ def __init__(self, drivers, width=0, height=0, coordMap=None,
         driver - instance that inherits from DriverBase
         width - X axis size of matrix
         height - Y axis size of matrix
-        coordMap - a 2D matrix defining the X,Y to strip index mapping. Not needed in most cases
-        rotation - how to rotate when generating the map. Not used if coordMap specified
+        coord_map - a 2D matrix defining the X,Y to strip index mapping. Not needed in most cases
+        rotation - how to rotate when generating the map. Not used if coord_map specified
```

Comment:
You're breaking 3.0.5 compat... I mean, yeah, no one probably uses it directly, just wanted to note it.